language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ServiceDeclarationNodeContext.java

```diff
@@ -134,7 +136,17 @@ public List<LSCompletionItem> getCompletions(BallerinaCompletionContext context,
             Eg:
             (1) service / m<cursor>
             function ...
+            (2) service TestObj m<cursor>
+            (3) service /<cursor>
+            (4) service / <cursor>
              */
+            if (!node.onKeyword().isMissing() ||
+                    context.getNodeAtCursor().textRange().endOffset() == context.getCursorPositionInTree() 
+                            && (context.getNodeAtCursor().toSourceCode().split(" ").length == 1
+                            && context.getNodeAtCursor().toSourceCode()
+                                .contains(SyntaxKind.SLASH_TOKEN.stringValue()))) {
```

Comment:
Let's move this condition to a separate method with a meaningful name like 'isInBaiscPathContext'.